macros/utils/table_operations/insert.sql

```diff
@@ -9,31 +9,52 @@
         {{ elementary.edr_log('Could not extract columns for table - ' ~ table_name ~ ' (might be a permissions issue)') }}
         {{ return(none) }}
     {% endif %}
-    {% set dicts_chunks = elementary.split_list_to_chunks(dicts, chunk_size) %}
-    {% for dicts_chunk in dicts_chunks %}
-        {% set insert_dicts_query = elementary.get_insert_dicts_query(table_relation, columns, dicts_chunk) %}
+    {% for dicts_chunk in dicts | batch(chunk_size) %}
```

Comment:
Not sure what's the benefit here in replacing the exsiting split_list_to_chunks to batch(chunk_size)?

Reply:
Why re-implement functionality that already exists natively in your runtime?

Reply:
It's just that it exists already and the logic is the same and batch is something that is not documented so I don't see a real value relying on it if we already have this logic. wdyt?